docs/datasync/soft-delete.md

```diff
@@ -0,0 +1,233 @@
+---
+id: soft-delete
+title: Soft Deletes with delta queries
+sidebar_label: Setting up Soft Deletion
+---
+
+This is the simplest strategy for Data Synchronization that is also quite straightforward to set up.
+
+## Setup
+
+Start off with the official Graphback template for MongoDB [*here*](https://GitHub.com/aerogear/graphback/tree/master/templates/ts-apollo-mongodb-backend) to follow along. Adding this strategy is relatively simple, consisting of the following two steps:
```

Comment:
I think we should refrain from linking to a file in a GitHub repo and show the user how to set up from the page itself, step-by-step.

Reply:
This could be part of the general enhancements we plan to do in datasync docs. 

/cc @wtrocki @ssd71

Reply:
We do have a datasync template now though, maybe we could do something with that?